pkg/grizzly/server.go

```diff
@@ -141,53 +147,71 @@ func (p *Server) Start() error {
 
 	r.Get("/api/live/ws", p.wsHandler)
 
-	if p.OpenBrowser {
-		var url string
+	if err := p.ParseResources(p.ResourcePath); err != nil {
+		return err
+	}
+
+	if p.openBrowser {
+		path := "/"
 
 		stat, err := os.Stat(p.ResourcePath)
 		if err != nil {
 			return err
 		}
 
-		if stat.IsDir() {
-			url = fmt.Sprintf("http://localhost:%d", p.Port)
-		} else {
-			resources, err := p.Parser.Parse()
+		if !stat.IsDir() && p.Resources.Len() == 0 {
+			return fmt.Errorf("no resources found to proxy")
+		}
+
+		if !stat.IsDir() && p.Resources.Len() != 0 {
```

Comment:
```suggestion
		if !stat.IsDir() && p.Resources.Len() == 1 {
```
Jsonnet can render multiple resources from a single file (or multiple resources can be in the same file). So we only want to open a specific resource in the browser if there's only one resource.